client/landing/stepper/declarative-flow/plugin-bundle-flow.ts

```diff
@@ -36,7 +36,7 @@ const getNextStep = ( currentStep: string, steps: StepperStep[] ): string | unde
 
 const SiteIntent = Onboard.SiteIntent;
 
-const pluginBundleFlow: Flow = {
+const pluginBundleFlow: FlowV1 = {
```

Comment:
Is there a particular reason we chose these flows and upgraded their type, or can it be done in a following PR if needed?

Reply:
Yes because these flows depend on `useAssertConditions` which doesn't exist in V2.